src/CONST.js

```diff
@@ -1078,6 +1078,29 @@ const CONST = {
         DEFAULT: 'en',
     },
 
+    LANGUAGES: ['en', 'es'],
```

Comment:
@BeeMargarida Maybe we could reuse `LOCALES` in your code instead of creating this one?

Reply:
They have different values, in this case we only want languages and not locales (not 'es_es' and 'es_es_onfido')